doc/guide/wire.md

```diff
@@ -0,0 +1,31 @@
+# What is Wire?
+
+Wire is a tool to deploy NixOS systems. Its configuration is a superset[^1] of [colmena](https://colmena.cli.rs/), however it is **not** a fork.
+
+[^1]: Any colmena configuration will continue to work with wire, but wire has additional ergonomic changes you can take advantage of.
+
+::: warning
+Wire is alpha software, please use at your own risk. Many features listed in this documentation may not be complete / implemented.
+:::
+
+<div class="tip custom-block" style="padding-top: 8px">
+
+Ready? Skip to the [Quickstart](./getting-started).
+
+</div>
+
+## Why Wire?
+
+::: info
+The following is the goal for a v1 release and not wholly implemented.
+:::
+
+| Features | Wire | Colmena |
+| -------------- | ------------- | -------- |
+| Keys & Secrets | :white_check_mark: | :white_check_mark: |
```

Comment:
```suggestion
| Secret management | :white_check_mark: | :white_check_mark: |
```

Something like this, keys & secrets is redundant phrasing as they're synonyms in this context.